### PR TITLE
Ar 197 arns reqs

### PR DESCRIPTION
**Pull request by ruudkas**
# Jira Ticket

Jira ticket number with link: [AR-197](https://academic-cs458.atlassian.net/jira/software/projects/AR/boards/1?selectedIssue=AR-197)

# Description of Changes
_Please describe what this pull request should accomplish_
Make it so you can use natural science to fill ARNS. Fixes ARNS for everything except for ABET for CS which would be way to complicated a change to the database to commit to at this point.
# Type of Change

- [x] Bug Fix
- [ ] New Feature
- [ ] Feature Update

# Checklist
- [x] My code follows ES Lint guidelines
- [x] My solution meets all of the acceptance criteria for the user story
- [x] My code contains necessary comments and documentation
- [x] I have added tests that are useful and cover the new changes
- [x] New and existing unit tests pass locally

### CODE REVIEW

File: components/FourYearPlanPage.tsx [hunk 1]
```diff
@@ -680,7 +680,6 @@ export const FourYearPlanPage: FC<FourYearPlanType> = memo(
         undo = false;
       }
     }
-
```
Comment (the10thWiz) [inline components/FourYearPlanPage.tsx:683]: Can we get this line back? It's quite nice for separation between the function body and the JSX return value.

**Review by the10thWiz — DISMISSED** [review]
Can we add tests to verify this?

**Review by Raffeln6264 — APPROVED** [review]
LGTM

**Comment by ruudkas** [comment]
> Can we add tests to verify this?

We now have tests

**Review by ChrisVasterling — APPROVED** [review]
lgtm

**Pull request merged**